WSDOTAgent/GirderScheduleChapterBuilder.cpp

```diff
@@ -683,6 +684,34 @@ rptChapter* CGirderScheduleChapterBuilder::Build(const std::shared_ptr<const WBF
       }
    }
 
+   Float64 tft = pIGirder->GetTopFlangeThickening(segmentKey);
+   if (!IsZero(tft))
+   {
+       GET_IFACE2(pBroker, IPointOfInterest, pPoi);
+       INIT_UV_PROTOTYPE(rptPointOfInterest, location, pDisplayUnits->GetSpanLengthUnit(), true);
+       INIT_UV_PROTOTYPE(rptLengthUnitValue, thickness, pDisplayUnits->GetComponentDimUnit(), true);
+
+       PoiList tfPoi;
+       pPoi->GetPointsOfInterest(segmentKey, POI_TENTH_POINTS | POI_SPAN, &tfPoi);
+
+       FlangeIndexType nFlanges = pIGirder->GetTopFlangeCount(girderKey);
+
+       (*pTable)(++row, 0) << _T("Top Flange Thickness:");
```

Comment:
The top flange thickening table should be a separate table from the girder schedule.

The table should mimic the Top Flange Thickness table from the standards drawing https://wsdot.wa.gov/publications/fulltext/Bridge/Web_BSD/5.6-A6-10-WFDG-UHPC-GIRDER-DETAILS-1-OF-5.PDF